rtengine/metadata.cc

```diff
@@ -312,6 +313,13 @@ void Exiv2Metadata::saveToImage(const Glib::ustring &path, bool preserve_all_tag
     }
 
     dst->exifData()["Exif.Image.Software"] = "RawTherapee " RTVERSION;
+
+    std::time_t t = std::time(nullptr);
+    char mbstr[100];
```

Comment:
This can be smaller

Reply:
`  char mbstr[20];`
  
  You mean like this, right?